### ISSUE TITLE
Is it possible to build the project on GNU/Debian 11 - 5.10.0-9-arm64 kernel (Raspberry Pi)?

### ISSUE THREAD

**Issue by blackmennewstyle** [ConcealNetwork/conceal-desktop#166]
Hello,

I would like to know if it is possible to build conceal-desktop on Raspberry Pi 4 8GB, running GNU/Linux Debian 11 with 5.10.0-9-arm64 kernel?

I'm getting the following error when trying to build:
```
mkdir -p build/release
cd build/release && cmake -D CMAKE_BUILD_TYPE=Release ../..
-- The C compiler identification is GNU 10.2.1
-- The CXX compiler identification is GNU 10.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Warning at CMakeLists.txt:29 (find_package):
  By not providing "FindQt5Charts.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "Qt5Charts", but CMake did not find one.

  Could not find a package configuration file provided by "Qt5Charts" with
  any of the following names:

    Qt5ChartsConfig.cmake
    qt5charts-config.cmake

  Add the installation prefix of "Qt5Charts" to CMAKE_PREFIX_PATH or set
  "Qt5Charts_DIR" to a directory containing one of the above files.  If
  "Qt5Charts" provides a separate development package or SDK, be sure it has
  been installed.


CMake Warning at /usr/share/cmake-3.18/Modules/FindBoost.cmake:1187 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1311 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1919 (_Boost_MISSING_DEPENDENCIES)
  CMakeLists.txt:39 (find_package)


CMake Warning at /usr/share/cmake-3.18/Modules/FindBoost.cmake:1187 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1311 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1919 (_Boost_MISSING_DEPENDENCIES)
  CMakeLists.txt:39 (find_package)


CMake Warning at /usr/share/cmake-3.18/Modules/FindBoost.cmake:1187 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1311 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1919 (_Boost_MISSING_DEPENDENCIES)
  CMakeLists.txt:39 (find_package)


CMake Warning at /usr/share/cmake-3.18/Modules/FindBoost.cmake:1187 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1311 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1919 (_Boost_MISSING_DEPENDENCIES)
  CMakeLists.txt:39 (find_package)


CMake Warning at /usr/share/cmake-3.18/Modules/FindBoost.cmake:1187 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1311 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1919 (_Boost_MISSING_DEPENDENCIES)
  CMakeLists.txt:39 (find_package)


CMake Warning at /usr/share/cmake-3.18/Modules/FindBoost.cmake:1187 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1311 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1919 (_Boost_MISSING_DEPENDENCIES)
  CMakeLists.txt:39 (find_package)


CMake Warning at /usr/share/cmake-3.18/Modules/FindBoost.cmake:1187 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1311 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1919 (_Boost_MISSING_DEPENDENCIES)
  CMakeLists.txt:39 (find_package)


CMake Warning at /usr/share/cmake-3.18/Modules/FindBoost.cmake:1187 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1311 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1919 (_Boost_MISSING_DEPENDENCIES)
  CMakeLists.txt:39 (find_package)


CMake Warning at /usr/share/cmake-3.18/Modules/FindBoost.cmake:1187 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1311 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1919 (_Boost_MISSING_DEPENDENCIES)
  CMakeLists.txt:39 (find_package)


-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
-- Found Boost: /usr/include (found suitable version "1.74.0", minimum required is "1.55") found components: date_time filesystem program_options regex serialization system thread chrono atomic 
-- Configuring done
-- Generating done
-- Build files have been written to: /home/ceedii/conceal-desktop/build/release
cd build/release && make
make[1]: Entering directory '/home/ceedii/conceal-desktop/build/release'
make[2]: Entering directory '/home/ceedii/conceal-desktop/build/release'
make[3]: Entering directory '/home/ceedii/conceal-desktop/build/release'
Scanning dependencies of target qrencode
make[3]: Leaving directory '/home/ceedii/conceal-desktop/build/release'
make[3]: Entering directory '/home/ceedii/conceal-desktop/build/release'
[  1%] Building C object CMakeFiles/qrencode.dir/libqrencode/bitstream.c.o
cc: error: unrecognized command-line option \u2018-maes\u2019
make[3]: *** [CMakeFiles/qrencode.dir/build.make:82: CMakeFiles/qrencode.dir/libqrencode/bitstream.c.o] Error 1
make[3]: Leaving directory '/home/ceedii/conceal-desktop/build/release'
make[2]: *** [CMakeFiles/Makefile2:157: CMakeFiles/qrencode.dir/all] Error 2
make[2]: Leaving directory '/home/ceedii/conceal-desktop/build/release'
make[1]: *** [Makefile:171: all] Error 2
make[1]: Leaving directory '/home/ceedii/conceal-desktop/build/release'
make: *** [Makefile:20: build-release] Error 2
```

**Comment by AxVultis** [comment]
Hello,
It is not possible at the moment to build conceal-desktop on Raspberry Pi.
It is only possible to build conceal-core for now.

**Comment by blackmennewstyle** [comment]
> Hello, It is not possible at the moment to build conceal-desktop on Raspberry Pi. It is only possible to build conceal-core for now.

Thanks for your answer. Are you guys working on it? Or is it not a priority for now?

**Comment by AxVultis** [comment]
Arm / Raspberry Pi support is available in development branch #172 and will be available for next release.
